app/src/main/resources/com/oracle/javafx/scenebuilder/app/i18n/SceneBuilderApp_ja.properties

```diff
@@ -394,10 +394,6 @@ alert.title.messagebox = 外部プログラムで開く
 
 alert.open.failure1.message = ''{0}''を開けませんでした
 alert.open.failure1.details = 開くのに失敗しました。選択されたファイルが有効なFXMLドキュメントであることを確認してください。
-alert.open.failureN.message = 指定されたファイルを開けませんでした
-alert.open.failureN.details = 開くのに失敗しました。それらのファイルが有効なFXMLドキュメントであることを確認してください。
-alert.open.failureMofN.message = {0}個のファイルを開けませんでした
-alert.open.failureMofN.details = いくつかのファイルを開くのに失敗しました。それらのファイルが有効なFXMLドキュメントであることを確認してください。
```

Comment:
with https://github.com/gluonhq/scenebuilder/pull/683 now merged, we have to do this for the chinese translation file as well.